index.bs

```diff
@@ -189,7 +189,7 @@ define ways to uniquely identify each one.
 
     <pre highlight="js">
     var sensor = new DirectTirePressureSensor({ position: "rear", side: "left" });
-    sensor.onreading = _ => console.log(sensor.pressure);
```

Comment:
This change would be even more helpful if it was just adding a new example, instead of changing the existing example.